contrib/pylightning/lightning/plugin.py

```diff
@@ -194,6 +194,9 @@ def _exec_func(self, func, request):
                 if pos < len(params):
                     # Apply positional args if we have them
                     arguments[k] = params[pos]
+                elif sig.parameters[k].default is inspect.Signature.empty:
+                    # This is a positional arg with no value passed
+                    return "Error - Missing required parameter: %s" % sig.parameters[k]
```

Comment:
```suggestion
                    raise TypeError("Error - Missing required parameter: %s" % sig.parameters[k])
```

Reply:
It should really raise an exception since otherwise we might end up using the `"Error - Missing required parameter: %s" % sig.parameters[k]` as a valid result from the call instead of detecting it as a failure.